WebApiSeed/Dtos/UpdateUserResultDto.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace WebApiSeed.Dtos
+{
+    /// <summary>
+    ///     Result of updating a user
+    /// </summary>
+    public class UpdateUserResultDto
+    {
+        /// <summary>
+        ///     User update result
+        /// </summary>
+        public UserUpdateResult Result { get; set; }
+
+        /// <summary>
+        ///     User dto
+        /// </summary>
+        public UserDto User { get; set; }
+    }
+}
```

Comment:
I think this Dto is not necesary the status can be handled directly by response status and yes be can return the USerDto directly